dvc/dirs.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import platformdirs
+
+APPNAME = "dvc"
+APPAUTHOR = "iterative"
+
+DVC_SYSTEM_CONFIG_DIR = "DVC_GLOBAL_CONFIG_DIR"
+DVC_GLOBAL_CONFIG_DIR = "DVC_GLOBAL_CONFIG_DIR"
+DVC_SITE_CACHE_DIR = "DVC_SITE_CACHE_DIR"
```

Comment:
We could have mocked it instead of creating envvars, but that won't work with tests where we run dvc through a subprocess. Like in celery, pre-commit hooks, etc.

Reply:
@skshetry Having them in `env.py` doesn't mean that they are public. Just put a comment there.

Reply:
Moved them to env.py.